src/remark-lint/no-heading-punctuation.ts

```
@@ -0,0 +1,45 @@
+import path from 'node:path'
+
+import type { Root } from 'mdast'
+import { toString } from 'mdast-util-to-string'
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)